deploy/deploy_app_server.yml

```diff
@@ -17,6 +17,8 @@
     - pycharm
     - nginx
     - nodejs
+    - redis
+    - scala
```

Comment:
@olivermel, I mean remove the "scala" role from the app server, can you do that?  This Pull Request doesn't include a Scala role, and so running it would fail.  You can leave "redis" on the Analytics Terminal too, doesn't hurt to have it there as well.